.github/issue_template.md

```diff
@@ -0,0 +1,2 @@
+<!-- This issue tracker is used only for tracking development events. -->
+<!-- If you want to create bug reports or feature requests, please go to the forums or the issue tracker on https://musescore.org/. -->
```

Comment:
Mind adding the direct links to the tracked and forums?

here they are for convenience:
[issue tracker](https://musescore.org/en/project/issues/musescore)
[forum](https://musescore.org/en/forum)

Reply:
The thing is, I don't want to redirect people only to the English forum. And yet, English forum is the place to go where most developers can see. So might as well just not link them and everyone choose their own forum.